src/organisms/Drawer/Drawer.tsx

```diff
@@ -106,25 +108,30 @@ export class DrawerContainer extends Component<DrawerProps, DrawerState> {
       footer,
     } = this.props;
     const { visible } = this.state;
+    if (visible && this.ref.current) {
+      this.ref.current.focus();
+    }
```

Comment:
Render methods should be pure (not calling side effects like focus) for performance optimizations. I'd rather move this to something like componentDidMount or componentDidMount. Alternatively, we can try using `<div autoFocus>` without the ref to focus it on its first render, if we prevent it from rendering until it's visible.

Reply:
I can move it out. Also hmmmm well the drawer does render but it just isn't visible until it slides in so that could cause a complication with using autoFocus I think.

Reply:
Ah, you think maybe we could block the first render until it's ready to be displayed? That's what I did for Tooltip.